go.mod

```diff
@@ -47,7 +47,7 @@ require (
 	github.com/hashicorp/vault/api v1.0.2
 	github.com/imdario/mergo v0.3.9 // indirect
 	github.com/influxdata/cron v0.0.0-20191203200038-ded12750aac6
-	github.com/influxdata/flux v0.112.1
+	github.com/influxdata/flux v0.113.0
```

Comment:
There's a line in `CHANGELOG.md` for the upgrade to `v0.112.1`, can you update it to ref `v0.113.0` and this PR instead?

Reply:
done